src/Illuminate/Foundation/Providers/ArtisanServiceProvider.php

```diff
@@ -88,6 +89,7 @@ class ArtisanServiceProvider extends ServiceProvider implements DeferrableProvid
         'ClearResets' => 'command.auth.resets.clear',
         'ConfigCache' => 'command.config.cache',
         'ConfigClear' => 'command.config.clear',
+        'Db' => DbCommand::class,
```

Comment:
Why is this different? may be `command.db`

Reply:
These are all going away in 9x. Refer https://github.com/laravel/framework/pull/34925